uwblueprint/jump-math
# (4) feat: display test sessions in classroom

## Notion ticket link

[View Test Sessions in Class](https://www.notion.so/uwblueprintexecs/View-Test-Sessions-in-Class-60c73526d09843fb8fecbb33fe2db609)

_Reviewing by commit might make the PR easier to review!_


## Implementation description
- [feat: update TestSessionListItem to support alternate design](https://github.com/uwblueprint/jump-math/pull/426/commits/dafb77a289fd7df5dcff0caa6c1ef4bde49ca5b1)
- [feat: add list items to test session table page](https://github.com/uwblueprint/jump-math/pull/426/commits/6a4c1c22c596ecc326b2daa2656ce2c13802e99d)
- [fix: padding in list-item-with-extra-elements](https://github.com/uwblueprint/jump-math/pull/426/commits/9b15af74e81bc8951fb2e5ca00025c2fa7c522d9)
- [feat: make assessments page sortable](https://github.com/uwblueprint/jump-math/pull/426/commits/1f5bb8b9dc5054ad6d2275904cf89d6df04b5f9d)
- [feat: make assessments page searchable](https://github.com/uwblueprint/jump-math/pull/426/commits/7a9a67250898d0d76e4dbf8f9f7b668bc60b6837)

<img width="1395" alt="image" src="https://github.com/uwblueprint/jump-math/assets/9922514/f6aa0a04-be98-4965-9c48-47565dcb03e8">




## Steps to test
1. View the class list page.
2. Click on a class card.
3. Click on the "Assessments" tab.
4. Verify you are redirected to a drilldown page with a populated, searchable, and sortable assessment list.



## What should reviewers focus on?
* 


## Checklist
